pkg/reconciler/kubernetes/tektonchain/transform.go

```diff
@@ -23,8 +23,18 @@ import (
 	"github.com/tektoncd/operator/pkg/apis/operator/v1alpha1"
 	"github.com/tektoncd/operator/pkg/reconciler/common"
 	"github.com/tektoncd/operator/pkg/reconciler/kubernetes/tektoninstallerset/client"
+	"knative.dev/pkg/ptr"
 )
 
+var defaultChainProperties v1alpha1.ChainProperties = v1alpha1.ChainProperties{
+	ArtifactsTaskRunFormat:      "in-toto",
+	ArtifactsTaskRunStorage:     ptr.String("oci"),
+	ArtifactsPipelineRunFormat:  "in-toto",
+	ArtifactsPipelineRunStorage: ptr.String("oci"),
+	ArtifactsOCIFormat:          "simplesigning",
+	ArtifactsOCIStorage:         ptr.String("oci"),
+}
```

Comment:
I see this values are added in the chains-config configMap, but since TektonConfig is the source of truth, I think these values should be reflected there as well

Reply:
You mean by default create a tektonConfig with these default, i think that make sens

Reply:
Yeah, since for user, TektonConfig is the source of truth